doc/manual/command-ref/conf-file.xml

```diff
@@ -865,6 +865,26 @@ requiredSystemFeatures = [ "kvm" ];
   </varlistentry>
 
 
+  <varlistentry xml:id="conf-tarball-ttl"><term><literal>tarball-ttl</literal></term>
+
+    <listitem>
+
+      <para>This option defines the number of seconds that a
+      downloaded tarball is cached in <filename>~/.cache/nix/tarballs/</filename>.
+      </para>
+
+      <para>This is most relevant for fetching tarballs that change remotely
+      via <envar>NIX_PATH</envar> or <function>fetchGit</function>.
+      Using <option>--option tarball-ttl 0</option> forces nix to re-fetch
+      these tarballs.</para>
```

Comment:
This is not enitirely correct. TTL=0 doesn't force a re-fetch, it forces Nix to check whether the tarball is still up-to-date using the ETag.

Reply:
I incorporated much of this text, and this feedback in https://github.com/NixOS/nix/pull/2882/files